api/python/cellxgene_census/pyproject.toml

```diff
@@ -40,9 +40,6 @@ dependencies= [
 
 [project.optional-dependencies]
 experimental = [
-    "torch",
-    "torchdata~=0.7,<0.10",  # FIXME: delete when the deprecated pytorch loader API prototype is fully graduated to TileDB-SOMA-ML
-    "scikit-learn>=1.2",
```

Comment:
While we're at it, should we remove `torchdata` from the intersphinx config?

https://github.com/chanzuckerberg/cellxgene-census/blob/99583e2d7deed4cde75a3d7915bb3396d7cacb81/docs/conf.py#L52-L54

I think `scikit-learn` and `torch` are still needed in intersphinx for the data loader notebook (which I think we're fine to keep overall since it uses the tiledb loader).

Though also intersphinx does not seem to be working in that notebook....